tests/test_hinfo.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+import sqlite3
+import tempfile
+from pathlib import Path
+
+import pytest
+import yaml
+from lsst.consdb import hinfo, utils
+from lsst.resources import ResourcePath
+from sqlalchemy import MetaData, Table, select
+
+
+@pytest.fixture
+def tmpdir(scope="module"):
+    os.environ["INSTRUMENT"] = "LATISS"
+    tmpdir = Path(tempfile.mkdtemp())
+    try:
+        yield tmpdir
+    finally:
+        shutil.rmtree(tmpdir)
```

Comment:
It can often be useful to specify `ignore_errors=True` when using `shutil.rmtree()`, especially when working on NFS filesystems that sometimes leave behind `.nfsNNNN` files when removing things.

In this case, since it's for temporary test files that should be on local storage, this is not an issue.